### ISSUE TITLE
Additions to add 

### ISSUE THREAD

**Issue by rosikand** [rosikand/matrix-calculus-cheatsheet#1]
Make a note of how the natural log ($\log_e$) is often used in ML loss functions. See lecture 1 derivation cs 224n.

**Comment by rosikand** [comment]
idea of applying vectorization to summation terms to pull certain variables out.

**Comment by rosikand** [comment]
log derivatives such as: 

$$\frac{d}{d x} \log_e [f(x)]=\frac{1}{f(x)} f^{\prime}(x)$$

**Comment by rosikand** [comment]
Make a new quarto component called "pattern" (e.g., the log cancelation def).

**Comment by rosikand** [comment]
Example to add: 
link to slide 32 from https://web.stanford.edu/class/cs224n/slides/cs224n-2023-lecture01-wordvecs1.pdf. Great derivation that covers lots of patterns.

**Comment by rosikand** [comment]
exp() chain rule

**Comment by rosikand** [comment]
what to do with summations

**Comment by rosikand** [comment]
examples: cross entropy deriv and softmax deriv and both (see [here](https://deepnotes.io/softmax-crossentropy)).

**Comment by rosikand** [comment]
vectorization of summation via transpose

**Comment by rosikand** [comment]
what to do when you have a $max(a, b)$ scenario and you need to take the deriv.

**Comment by rosikand** [comment]
Add hinge loss derivative from cs 221 midterm
